kratos/tests/cpp_tests/geometries/test_quadrature_point_geometry.cpp

```diff
@@ -74,6 +74,35 @@ namespace Kratos {
             return p_this_quadrature_point;
         }
 
+        QuadraturePointGeometry<Node<3>, 2, 2>::Pointer GenerateQuadraturePointGeometry2() {
```

Comment:
is it intentional that you often use by-value and not by-reference for Matrix and Vector in this function?

Reply:
I am not sure if it would be safe to do so...

Reply:
We are actually talking about DN_De, aren't we?

Reply:
yes
Ok if you are not sure, just asking